lib/modules.coffee

```diff
@@ -5,17 +5,38 @@ module.exports =
   activate: ->
     @createStatusUI()
 
-    @activeItemSubscription = atom.workspace.onDidChangeActivePaneItem =>
-      @update()
-    client.onConnected => @update()
-    client.onDisconnected => @update()
-    @update()
+    # configure all the events that might require setting or clearing the
+    # cursor move callback
+    atom.workspace.onDidChangeActivePaneItem => @activePaneChanged()
```

Comment:
You'll want to keep the `@activeItemSubscription =` part here so that it gets disposed of properly.